src/bootstrap/config.rs

```diff
@@ -818,7 +824,6 @@ impl Config {
                 check_ci_llvm!(llvm.targets);
                 check_ci_llvm!(llvm.experimental_targets);
                 check_ci_llvm!(llvm.link_jobs);
-                check_ci_llvm!(llvm.link_shared);
```

Comment:
Hm, was this dropped by accident perhaps? It seems like the check here is still necessary; we don't want users specifying this manually if they're using CI LLVM.

Reply:
This line is a duplicate, it's already checked above on line 822: https://github.com/rust-lang/rust/blob/5729f875a7c691e2043621a9b05b9fe0eede1630/src/bootstrap/config.rs#L822